x-pack/test/observability_functional/apps/observability/pages/rules_page.ts

```diff
@@ -208,6 +208,14 @@ export default ({ getService, getPageObjects }: FtrProviderContext) => {
 
         const consumerSelect = await testSubjects.find('ruleConsumerSelection');
         await consumerSelect.click();
+
+        await retry.waitFor('consumer options to have text', async () => {
```

Comment:
Is this related to seeing the setup guide page, as you shared [here](https://github.com/elastic/kibana/issues/214239#issuecomment-3020132895)?